package.json

```diff
@@ -29,7 +34,10 @@
   ],
   "types": "kleur.d.ts",
   "devDependencies": {
-    "tap-spec": "^5.0.0",
-    "tape": "^4.9.1"
+    "bundt": "1.0.0",
+    "tap": "14.10.7"
+  },
+  "exports": {
+    ".": "./index.mjs"
```

Comment:
I'm 99% certain that we still have to differentiate between `import` and `require` types here. 
Node 14 will look at the `exports` map since that's what it wants to read, and will throw if there's no `require` target. The `main` is preserved & relevant for non-`exports` runtimes.